src/components/MessageListMessageCompact.vue

```diff
@@ -43,6 +43,11 @@
         >
             {{ ml.formatTime(message.time) }}
         </div>
+        <span
```

Comment:
This div is used in a few places. It might be better to create a component for it so it's not duplicated all over the place